Exception catched: the dataframe went out of scope when booking a filter
